### ISSUE TITLE
Hugo benchmark Prepare pages failed

### ISSUE THREAD

**Issue by anagio** [gohugoio/hugo#4033]
Using Hugo 3.0.2 I'm getting

`Error: Prepare pages failed: ["html/template: cannot Parse after Execute"]…` after the first site is built. I also tested the benchmark command with 0.19

**Comment by bep** [comment]
You say that `hugo benchmark` fails?

I just tested this OK on a couple sites. Which means there must be something special with your site. Do you have some source to share?

**Comment by anagio** [comment]
Verbose output only has one warning about an empty index being rendered. No other warnings or errors. I'm not able to share source for this project, anything else I can share to help debug?

**Comment by bep** [comment]
>I'm not able to share source for this project, anything else I can share to help debug?

That is the only way I can help you, I'm afraid. I will have to close this for now.

**Comment by anagio** [comment]
FWIW, I searched the source for the error message and found https://github.com/gohugoio/hugo/blob/b5e1dc5892f81da798d0d4e964a1f3328532f45e/tpl/tplimpl/template.go#L185

I then checked my content folder and found I had some content files uncommitted which I was testing locally without a matching template. So Hugo worked as expected and gave me that error while running benchmark. Confusing though because the first run of builds works with benchmark the second run is where it failed.
